client/src/components/TextInput.tsx

```diff
@@ -48,7 +53,14 @@ const StyledInput = styled.input`
   }
 `;
 
-function TextInput({ label, row, col, ...props }) {
+type Props = {
+  label: string;
+  row: number | string;
```

Comment:
why are we using a union here? like is there a reason for it to be a number or a string?

Reply:
rows/columns can be numbers or they can be ranges like 2/4 which would be represented as a string